src/Web/Microsoft.NET.Sdk.Web.ProjectSystem.Targets/netstandard1.0/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -43,5 +43,13 @@ Copyright (c) .NET Foundation. All rights reserved.
       <Context>BrowseObject</Context>
     </PropertyPageSchema>
   </ItemGroup>
-
-</Project>
\ No newline at end of file
+  
+  <!-- Web specific target to ensure that generated content (css, js, etc.) is copied to output -->
+  <Target Name="_IncludeWebProjectGeneratedContent" BeforeTargets="GetCopyToPublishDirectoryItems" Condition=" '$(EnableDefaultItems)' == 'true' And '$(EnableDefaultContentItems)' == 'true' ">
```

Comment:
BeforePublish & AfterPublish extensibility should work even now from both VS & CLI. There is an empty "BeforePublish" & "AfterPublish" target in websdk that might be overriding other custom targets but the below targets for example should just work from both VS & CLI.
```xml
<Target Name="CustomActionsBeforePublish" BeforeTargets="BeforePublish">
    <Message Text="Actions BeforePublish" Importance="high" />
  </Target>
  <Target Name="CustomActionsAfterPublish" AfterTargets="AfterPublish">
    <Message Text="Actions AfterPublish" Importance="high" />
  </Target>
```
If you are running into any issues with these feel free to open bugs.